<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_movie</class>
 <widget class="QWidget" name="add_movie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>174</width>
    <height>317</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>180</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление фильма</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(89, 108, 134);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="9" column="0">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Режиссер:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Жанр:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Цена билета:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Драма</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Комедия</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Экшн</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ужасы</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Фэнтези</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Романтика</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Мистика</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Анимация</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Криминал</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Год выпуска:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Добавление фильма</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
